### PR TITLE
ansible rosa draft v0.2

### PR DESCRIPTION
**Pull request by dsari-rh**

### CODE REVIEW

File: content/docs/rosa/ansible-rosa/index.md [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+
+To help with DNS resolution, we will be using DNS forwarder to forward the queries to [Route 53 Inbound Resolver](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/resolver.html) allowing the cluster to accept incoming DNS queries from external sources and thus establishing the desired connection without exposing the underlying infrastructure. 
+
+![ROSA with PL and TGW](images/rosa-pl-tgw-newicons.png)
+
+In addition, [Egress VPC](https://docs.aws.amazon.com/managedservices/latest/onboardingguide/networking-vpc.html) will be provisioned serving as a dedicated network component for managing outbound traffic from the cluster. A [NAT Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html) will be created within the public subnet of the Egress VPC and along with it a [Squid](http://www.squid-cache.org/)-based proxy to restrict egress traffic from the cluster to only the permitted endpoints or destinations.
```
Comment (paulczar) [inline content/docs/rosa/ansible-rosa/index.md:20]: link to the list of required endpoints in the rosa docs.

File: content/docs/rosa/ansible-rosa/index.md [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+jumphost_instance_type: t2.micro
+
+proxy_enabled: true
+```
+
+Next, we will talk about what <code>make</code> is and how it helps compiling the code for our deployment in this scenario.
+
+
+### Make
+Make is a build automation tool to manage the compilation and execution of programs. It reads a file called a "makefile" that contains a set of rules and dependencies, allowing developers to define how source code files should be compiled, linked, and executed.
```
Comment (paulczar) [inline content/docs/rosa/ansible-rosa/index.md:118]: Makefile

**Review by paulczar — CHANGES_REQUESTED** [review]

File: content/docs/rosa/ansible-rosa/index.md (hunk 1 above)
Comment (dsari-rh) [reply content/docs/rosa/ansible-rosa/index.md:20]: done

File: content/docs/rosa/ansible-rosa/index.md [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+
+Finally, once the cluster is created, we will access it by establishing secure SSH connection using a jump host that is set up within the Egress VPC, and to do so we will be using [sshuttle](https://sshuttle.readthedocs.io/en/stable/). 
+
+
+### Git
+Git is version control system that tracks changes to files and enables collaboration, while GitHub is a web-based hosting service for Git repositories. And in this scenario, the deployment will be based on the Ansible playbook from MOBB GitHub repo at [https://github.com/rh-mobb/ansible-rosa](https://github.com/rh-mobb/ansible-rosa). 
+
+We are specifying the environment and variables in the following directories:
+* <code>./environment/*/group_vars/all.yaml</code> - environment setup
+* <code>./roles/_vars/defaults/main.yml</code> - variables
```
[the remark this answers is not in the document]
Comment (dsari-rh) [reply content/docs/rosa/ansible-rosa/index.md:32]: done

File: content/docs/rosa/ansible-rosa/index.md (hunk 2 above)
Comment (dsari-rh) [reply content/docs/rosa/ansible-rosa/index.md:118]: done

File: content/docs/rosa/ansible-rosa/index.md [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+
+
+### Make
+Make is a build automation tool to manage the compilation and execution of programs. It reads a file called a "makefile" that contains a set of rules and dependencies, allowing developers to define how source code files should be compiled, linked, and executed.
+
+In this scenario, the makefile can be found in the root directory of the GitHub repo, and here below is the snippet where the cluster name is set up along with the virtual environment that makefile will compile when we are running <code>make virtualenv</code>: 
+
+```bash
+CLUSTER_NAME ?= ans-$(shell whoami)
+EXTRA_VARS ?= --extra-vars "cluster_name=$(CLUSTER_NAME)"
```
[the remark this answers is not in the document]
Comment (dsari-rh) [reply content/docs/rosa/ansible-rosa/index.md:124]: done

**Review by paulczar — APPROVED** [review]

**Pull request merged**